unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -323,6 +322,11 @@ public void ConfigureHUDElement(HUDElementID hudElementId, HUDConfiguration conf
             hudElement.SetVisibility(configuration.active && configuration.visible);
     }
 
+    private HUDConfiguration ConfigurationFromJson(HUDElementID hudElementId, string configuration)
+    {
+        return JsonUtility.FromJson<HUDConfiguration>(configuration);
+    }
```

Comment:
Why we have to pass the `hudElementId` parameter in this function?

Reply:
Will mark this as resolved. I'll be implementing the new approach we are discussing on slack